feature/mpls/otg_tests/label_block/README.md

```diff
@@ -0,0 +1,58 @@
+# MPLS-1.1: MPLS label blocks using ISIS
+
+## Summary
+
+Define reserved MPLS label blocks: static and MPLS-SR.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+Topology: ATE1—DUT1
+
+On DUT1 configure:
+
+*   ISIS adjacency between ATE1 and DUT1
+*   Enable MPLS-SR
```

Comment:
Please add some additional explanation of how to do this.  I don't see a path that 1:1 maps to mpls enable segment routing?

Reply:
Added the path.